R/styler.R

```diff
@@ -7,7 +7,8 @@
 #'  * [style_file()] to style a single .R file.
 #'  * [style_dir()] to style all .R files in a directory.
 #'  * [style_pkg()] to style the source files of an R package.
-#'  * An RStudio Addin to style the active file .R file and the current package.
+#'  * [styler_addins] (RStudio Addins) to style either selected code or the
```

Comment:
Can you add two braces after `styler_addins` (as it is done with `style_pkg()` etc. above) to not just make a reference, but also use code font markup?

Reply:
I was deliberately trying to avoid making the reference look like a function since it's not. Does that distinction not really matter here?

Reply:
Ok, that actually makes sense. You are right.